eras/conway/impl/cardano-ledger-conway.cabal

```diff
@@ -63,10 +68,14 @@ library
                 cardano-ledger-mary,
                 cardano-ledger-shelley,
                 containers,
+                data-default,
+                data-default-class,
```

Comment:
No need to depend on both. `Data.Default` also exports the `Default` class. Rule of thumb is: depend on `data-default` if any of the instances from the package are needed, otherwise depend on `data-default-class`. I can't tell if we need `data-default` here, but I am sure you'll know which one should be used.
```suggestion
```